packages/core/lib/modules/accounts.test.js

```diff
@@ -0,0 +1,48 @@
+import {Accounts} from './accounts';
+import {Wallet} from './wallet';
+
+describe('Accounts module', () => {
```

Comment:
I really want to see high coverage here including error flows (e.g. invalid parameters, duplicate accounts, etc.)

Reply:
Yes, will be handled in an additional PR. https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871